.gitmodules

```diff
@@ -2,3 +2,9 @@
 	path = wiki
 	url = https://github.com/autosarzs/Dev.wiki.git
 	branch = master
+[submodule "CMock"]
```

Comment:
I believe that the duplication of submodule was done by mistake, could you please confirm?

Reply:
yes, it was done by mistake